doc/admin-guide/logging/formatting.en.rst

```diff
@@ -628,6 +629,7 @@ cqssv  Client Request SSL version used to communicate with the client.
 cqssc  Client Request SSL Cipher used by |TS| to communicate with the client.
 cqssu  Client Request SSL Elliptic Curve used by |TS| to communicate with the
                       client when using an ECDHE cipher.
+cqssa  Client Request ALPN Protocol ID negotiated with the client.
```

Comment:
Ins't it almost the same as `cqpv`? Do we need the both?

Reply:
Yes, please look at the updated description.